slackcollector/collector.py

```diff
@@ -74,10 +61,23 @@ def load_config(self, config_file_path):
         """Parse the configuration file."""
         config_file = os.path.join(os.path.dirname(__file__),
                                    '../config/' + config_file_path)
-        config = yaml.safe_load(open(config_file))
-        self.data_dir = config['storage']['data_dir']
-        self.data_file_prefix = config['storage']['data_file_prefix']
-        slack.api_token = config['secure']['slack_group_token']
+        try:
+            config = yaml.safe_load(open(config_file))
+            self.data_dir = config['storage']['data_dir']
+            self.data_file_prefix = config['storage']['data_file_prefix']
+            slack.api_token = config['secure']['slack_group_token']
+        except IOError:
+            self.logger.error(
+                'Configuration file does not exist. Make sure'
+                ' the file "{}" exists and is '
+                'configured correctly.'.format(config_file))
+            sys.exit(1)
+        except yaml.scanner.ScannerError:
+            self.logger.error(
```

Comment:
@joehakimrahme wondering why this is done manually here, can't the file loader check for syntax sanity depending on format?